src/Xamarin.Android.Build.Tasks/Tasks/CopyIfChanged.cs

```diff
@@ -24,11 +24,18 @@ public class CopyIfChanged : AndroidTask
 		[Required]
 		public ITaskItem[] DestinationFiles { get; set; }
 
+		public bool CompareFileLengths { get; set; }
```

Comment:
```suggestion
		public bool CompareFileLengths { get; set; } = true;
```
Then you could remove that new ctor.

Reply:
I tried that but it didn't work for some reason...